mappings/net/minecraft/client/render/entity/ItemRenderer.mapping

```diff
@@ -1,4 +1,30 @@
 CLASS net/minecraft/class_92 net/minecraft/client/render/entity/ItemRenderer
 	FIELD field_1709 rand Ljava/util/Random;
+	METHOD method_1483 (IIIIII)V
+		ARG 1 x
+		ARG 2 y
+	METHOD method_1485 fillTessellator (Lnet/minecraft/class_67;IIIII)V
```

Comment:
I read this message name and have no clue what this method does? make a rectangle?

Reply:
yes it fills the tessellator with vertices its private noone cares :tm: